android/sdl_android/src/androidTest/java/com/smartdevicelink/test/Validator.java

```diff
@@ -2385,6 +2385,10 @@ public static boolean validatePermissionItem(PermissionItem item1, PermissionIte
             return false;
         }
 
+        if ( item1.getEncryptionRequirement() != item2.getEncryptionRequirement()) {
+        	return false;
+		}
```

Comment:
Very minor request, could we please match the spacing from the if statement above?

Reply:
Done